Modix.Data/Repositories/DateTimeOffsetRange.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Modix.Data.Repositories
```

Comment:
Should this be here?

Reply:
As in, is this the appropriate namespace for this class?

The class, at the moment, only exists for the sake of searching through data records, which is a Repository-level concern, (and this, is only referenced by repositories). I wouldn't want to put it in Modix.Data.Models, as it's not a Model type to be stored in the DB.

Where would you put it?

Reply:
In a common library or something like an extensions folder. This could be used elsewhere, other than at repository level.